internal/providers/vsphere/pke/driver/cluster_updater.go

```diff
@@ -0,0 +1,416 @@
+// +build 0
+
+// Copyright © 2019 Banzai Cloud
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package driver
+
+import (
+	"context"
+	"net"
+	"net/http"
+	"time"
+
+	"emperror.dev/errors"
+	"github.com/sirupsen/logrus"
```

Comment:
Do not use logrus. Use the common logger interface. (Can be part of a subsequent PR)